code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -192,6 +192,8 @@ mob/living/simple_animal/hostile/hitby(atom/movable/AM as mob|obj,var/speed = TH
 		if(HOSTILE_STANCE_IDLE)
 			targets = ListTargets(10)
 			target_mob = FindTarget()
+			if(isnull(target_mob))

```

Comment:
There is no destroy_surroundings check on this. That means that mobs that aren't supposed to destroy surroundings, like passive holocarp and drones, will destroy surroundings.

Reply:
Great point